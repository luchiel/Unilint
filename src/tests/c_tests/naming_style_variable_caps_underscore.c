/*
style ordering:
    underscore
    pascal
    camel
    caps_underscore
    invalid
*/

//structs
struct abc_abc { int A; };
struct AbcAbc { int A; };
struct abcAbc { int A; };
struct ABC_ABC { int A; };
struct Abc_abC { int ABC_ABC; };

//functions
int qwerty_1() { return 1; }
int Qwerty2() { return 2; }
int qwertyThree() { return 3; }
int QWERTY_FOUR() { return 4; }
int Querty_5() { return 5; }

//typedefs
typedef sabaka_ int;
typedef SabakA int;
typedef sabakA int;
typedef SABAKA_ int;
typedef SaB_aKa int;

//variables
int avada_kedavra;
int avadaKedavra;
int AvadaKedavra;
int AVADA_KEDAVRA;
int 111avada_kenTavra;