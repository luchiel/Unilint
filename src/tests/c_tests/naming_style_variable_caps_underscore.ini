[naming_styles]
variable_naming_style = caps_underscore