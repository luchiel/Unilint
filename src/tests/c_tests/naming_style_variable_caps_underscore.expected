32:5: name avada_kedavra is not of style caps_underscore
33:5: name avadaKedavra is not of style caps_underscore
34:5: name AvadaKedavra is not of style caps_underscore
36:5: name 111avada_kenTavra is not of style caps_underscore
